.github/actions/upload-assets/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "upload-release-asset",
```

Comment:
upload-assets? or maybe rename the action's directory to `upload-release-asset`?

Reply:
will do thanks